doc/source/whatsnew/v0.21.0.txt

```diff
@@ -91,6 +91,8 @@ Bug Fixes
 
 Conversion
 ^^^^^^^^^^
+- Fix a bug when pd.astype() receive Series as `dtype` paramter, no action is taken. Now Series 
```

Comment:
``Bug in :func:`DataFarme.astype` where a passed Series as a dtype mapping would be ignored``